rotkehlchen/db/dbhandler.py

```diff
@@ -1337,6 +1339,15 @@ def save_tokens_for_address(
             insert_rows,
         )
 
+    def get_evm_accounts(self, cursor: 'DBCursor') -> list[ChecksumEvmAddress]:
+        """Returns a list of unique EVM accounts from all EVM chains."""
+        placeholders = ','.join('?' * len(SUPPORTED_EVM_CHAINS))
+        cursor.execute(
+            f'SELECT DISTINCT account FROM blockchain_accounts WHERE blockchain IN ({placeholders});',  # noqa: E501
+            [chain.value for chain in SUPPORTED_EVM_CHAINS],
+        )
+        return [to_checksum_address(entry[0]) for entry in cursor]
```

Comment:
You don't need to do that. `checksum_to_address` is expensive. Since we always save checksummed addresses in the DB you can assume they already are

```suggestion
        return [entry[0] for entry in cursor]
```